src/style.css

```
@@ -0,0 +1,66 @@
+.body {
+  margin: 0;
+  background-color: white;
+}
+
+.body--child {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+  height: 16px;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  justify-content: space-between;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}
```
and you can do it like this